core/src/test/java/org/springframework/security/oauth2/server/authorization/InMemoryOAuth2AuthorizationServiceTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.server.authorization;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.security.oauth2.server.authorization.InMemoryOAuth2AuthorizationService.TokenWithType;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for {@link InMemoryOAuth2AuthorizationService}.
+ *
+ * @author Krisztian Toth
+ */
+public class InMemoryOAuth2AuthorizationServiceTest {
```

Comment:
Please take a look at `InMemoryRegisteredClientRepositoryTests` and `InMemoryOAuth2AuthorizedClientServiceTests` (`spring-security-oauth2-client` module) to see how our tests are written.

The main points are:
- use AssertJ - not JUnit's assert
- use Mockito for mocking expectations
- the naming conventions for methods name follow a pattern

Please apply the changes here to align with the coding conventions/style that Spring Security uses.

Reply:
Sorry for that, I should have checked. Updated in 48cc835